<?xml version="1.0"?>
<launch>
    <arg name="target_dist_to_goal" default="0.5"/>
    <arg name="path_topic" default="/path"/>
    <arg name="robot_pose_topic" default="/robot_pose"/>
    <node pkg="local_goal_creator_ros" type="local_goal_creator_node" name="local_goal_creator" output="screen">
        <param name="target_dist_to_goal" value="$(arg target_dist_to_goal)"/>
        <remap from="/path" to="$(arg path_topic)"/>
        <remap from="/robot_pose" to="$(arg robot_pose_topic)"/>
    </node>
</launch>
